crates/cheatnet/src/state.rs

```diff
@@ -158,6 +159,24 @@ pub struct CallTrace {
     pub nested_calls: Vec<Rc<RefCell<CallTrace>>>,
     pub result: CallResult,
     pub vm_trace: Option<Vec<TraceEntry>>,
+    pub node_type: NodeType,
+}
+
+#[derive(Default, Copy, Clone)]
+pub enum NodeType {
+    #[default]
+    Regular,
+    Phantom,
+}
```

Comment:
Probably should be added to the profiler definition, with some comments why we need those nodes to be present.

Reply:
It is added to the profiler definition